### PR TITLE
🇨🇿 Adding metadata for the Czech Republic in integration

### PR DESCRIPTION
**Pull request by CharlesIC**

### CODE REVIEW

**Review by Wynndow — APPROVED** [review]
I'd never heard of "Czechia" before....apparently it's not popular! https://www.theguardian.com/world/2016/oct/25/nobody-calls-it-czechia-czech-republic-new-fails-catch-on

**Review by JohnWattsGDS — APPROVED** [review]

**Comment by CharlesIC** [comment]
@Wynndow 
Haha yeah I've seen that article. It is the official name adopted by the Czech Republic, however.... Even Google Maps calls it Czechia! I think @jakubmiarka doesn't like it, though 😅
I just used it cos it's shorter 🙃

File: src/main/resources/integration/MetadataSourceConfiguration.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "metadataUrls": {
     "Belgium metadata": "https://idp.iamfas.qa.belgium.be/EidasNode/ServiceMetadata",
+    "Czechia metadata": "https://srv.dev.eidasnode.cz/EidasNode/ServiceMetadata",
```
Comment (jakubmiarka) [inline src/main/resources/integration/MetadataSourceConfiguration.json]:
```suggestion
    "Czech Republic metadata": "https://srv.dev.eidasnode.cz/EidasNode/ServiceMetadata",
```

Comment (jakubmiarka) [reply src/main/resources/integration/MetadataSourceConfiguration.json]: 😆

Comment (Wynndow) [reply src/main/resources/integration/MetadataSourceConfiguration.json]: ha!

**Comment by JohnWattsGDS** [comment]
> I just used it cos it's shorter 🙃

How often do you plan on typing it @CharlesIC ?

File: src/main/resources/integration/MetadataSourceConfiguration.json (hunk 1 above)
Comment (CharlesIC) [reply src/main/resources/integration/MetadataSourceConfiguration.json]: "Czech Republic" is too long, Jakub 😝

**Comment by CharlesIC** [comment]
Of course Wikipedia has a whole page dedicated to the issue of the name of the country of the Czech Republic... 😅
https://en.wikipedia.org/wiki/Name_of_the_Czech_Republic

"Czechia" is the official name used at the UN so I see no issue with using it.

The official UK country register lists "Czechia" as the name:
https://country.register.gov.uk/records/CZ.json
```
official-name | "The Czech Republic"
name | "Czechia"

```

Compare to the UK:
https://country.register.gov.uk/records/GB.json
```
official-name | "The United Kingdom of Great Britain and Northern Ireland"
name |  "United Kingdom"
```

And the case is the same for many other countries - say "The Italian Republic" and "Italy" in the register.

We use the short version of country names so I feel it's legit to use "Czechia" too 🤷‍♂

**Pull request merged**